probe_scraper/scrapers/moz_central_scraper.py

```diff
@@ -181,6 +181,8 @@ def download_files(channel, node, temp_dir, error_cache, version, tree=None):
     if tree is None:
         uri = CHANNELS[channel]['base_uri']
     else:
+        if not tree.startswith("releases/") and tree != "mozilla-central":
```

Comment:
This issue occurred locally but not on prod - for some reason, sometimes buildhub returns `mozilla-beta` or `mozilla-release` with `releases/` prefixed, but sometimes not.

Reply:
Woha. That's weird. Would you kindly add your comment before this line ?